### PR TITLE
docs: catalog-info.yaml should primarily use "Web" for icon

### PR DESCRIPTION
**Pull request by sarina**
Addresses #568

### CODE REVIEW

File: oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst [hunk 1]
```diff
@@ -114,9 +114,8 @@ References
       links:
         - url: "https://docs.openedx.org"
           title: "Deployed Site"
-          # Backstage uses the MaterialUI Icon Set.
-          # https://mui.com/material-ui/material-icons/
-          # The value will be the name of the icon.
+          # Backstage uses the MaterialUI Icon Set (https://mui.com/material-ui/material-icons/)
+          # The value will be the name of the icon. We recommend using "Web" for all icons.
```
Comment (robrap) [inline oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst:118]: If we want people to have the flexibility of choosing something more appropriate for certain repos, we could make this small change.
```suggestion
          # The value will be the name of the icon. We recommend using "Web".
```

File: oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst [hunk 2]
```diff
@@ -114,9 +114,8 @@ References
       links:
         - url: "https://docs.openedx.org"
           title: "Deployed Site"
-          # Backstage uses the MaterialUI Icon Set.
-          # https://mui.com/material-ui/material-icons/
-          # The value will be the name of the icon.
+          # Backstage uses the MaterialUI Icon Set (https://mui.com/material-ui/material-icons/)
```
Comment (robrap) [inline oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst:117]: Nit:
```suggestion
          # Backstage uses the Material UI Icon Set (https://mui.com/material-ui/material-icons/)
```

**Review by robrap — APPROVED** [review]
Thank you! I provided minor possible changes.

**Review by robrap — COMMENTED** [review]
Also, do we want to add a Change History section?

**Review by feanil — COMMENTED** [review]
Suggestion to take or leave.  I like my version because it makes it clear that it sometimes makes sense to use other icons but mostly you should stick to "Web"

**Comment by sarina** [comment]
@feanil I agree, your wording captures it well

File: oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst [hunk 3]
```diff
@@ -114,9 +114,8 @@ References
       links:
         - url: "https://docs.openedx.org"
           title: "Deployed Site"
-          # Backstage uses the MaterialUI Icon Set.
-          # https://mui.com/material-ui/material-icons/
-          # The value will be the name of the icon.
+          # Backstage uses the Material UI Icon Set (https://mui.com/material-ui/material-icons/)
+          # The value will be the name of the icon. Use "Web" unless there is a more appropriate icon.```
```
Comment (feanil) [inline oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst:118]:
```suggestion
          # The value will be the name of the icon. Use "Web" unless there is a more appropriate icon.
```

Removing some errant backticks.

**Review by feanil — APPROVED** [review]

**Pull request merged**